src/modules/notify/connectors/scheduled-notification.js

```diff
@@ -5,13 +5,11 @@ const scheduledNotification = require('../../../controllers/notifications').repo
 const snakeCaseKeys = require('snakecase-keys')
 const { findOne } = require('../../../lib/repository-helpers')
 
-const stringifyArray = (value) => Array.isArray(value) ? JSON.stringify(value) : value
-
 const mapObjectToNotification = (data) => {
   const snakeCaseData = snakeCaseKeys(data)
   const dataStringify = {}
   for (const data in snakeCaseData) {
-    dataStringify[data] = stringifyArray(snakeCaseData[data])
+    dataStringify[data] = Array.isArray(snakeCaseData[data]) ? JSON.stringify(snakeCaseData[data]) : snakeCaseData[data]
   }
```

Comment:
Looking at how we're iterating in `src/modules/batch-notifications/config/returns/lib/get-recipients.js` I suspect we could write this to work the same way:
```js
  for (const [key, value] of Object.entries(snakeCaseData)) {
    dataStringify[key] = Array.isArray(value) ? JSON.stringify(value) : value
  }
```